packages/http/src/__tests__/http-prism-instance.spec.ts

```diff
@@ -1,114 +1,268 @@
 import { IPrism } from '@stoplight/prism-core';
+import { DiagnosticSeverity } from '@stoplight/types';
 import { IHttpOperation } from '@stoplight/types/http-spec';
+import * as nock from 'nock';
 import { relative, resolve } from 'path';
 import { createInstance, IHttpConfig, IHttpRequest, IHttpResponse, ProblemJsonError } from '../';
 import { forwarder } from '../forwarder';
 import { UNPROCESSABLE_ENTITY } from '../mocker/errors';
-import { NO_PATH_MATCHED_ERROR } from '../router/errors';
+import { NO_BASE_URL_ERROR, NO_PATH_MATCHED_ERROR, NO_SERVER_MATCHED_ERROR } from '../router/errors';
+
+const fixturePath = (filename: string) => relative(process.cwd(), resolve(__dirname, 'fixtures', filename));
```

Comment:
I think the relative part is superfluous, the __dirname is an absolute path so you can just resolve(__dirname, ...